webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -13,13 +13,15 @@
         <hc-user :user="currentUser" :trunc="35" />
         <ds-space />
         <ds-input model="title" class="post-title" placeholder="Title" name="title" autofocus />
+        <small class="smallTag">{{ form.title.length }}/{{ formSchema.title.max }}</small>
         <no-ssr>
           <hc-editor
             :users="users"
             :hashtags="hashtags"
             :value="form.content"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-09T20:23:29Z" title="Tuesday, July 9th 2019, 10:23:29 pm +02:00">Jul 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -169,6 +169,16 @@ export default {
     },
     updateEditorContent(value) {
       // this.form.content = value
+      this.disabled = true
+      if (
+        value.replace(
+          /<p>|<\/p>|<h3>|<\/h3>|<h4>|<\/h4>|<ul>|<\/ul>|<ol>|<\/ol>|<li>|<\/li>|<blockquote>|<\/blockquote>|<strong>|<\/strong>|<em>|<\/em>|<a.*>|<\/a>/gm,
```


Please don't use a regex. If you want to strip HTML from a string, the [internet has the answer](https://stackoverflow.com/questions/5002111/how-to-strip-html-tags-from-string-in-javascript).. Regexes are horrendously hard to maintain. "If you found out you can solve your problem with a regex, you have a new problem."

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-07-10T04:45:46Z" title="Wednesday, July 10th 2019, 6:45:46 am +02:00">Jul 10, 2019</time>_

---


to count the absolute characters, a lenght function is not enough. because the editor just inserts these html tags. so i only count the real characters that a user writes. nothing will be deleted or added from the original text. 

since I hold myself back from inserting npm packete. 

or I don't know if this already exists. 

since i love regular expressions that was my solution :)

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-07-10T05:49:04Z" title="Wednesday, July 10th 2019, 7:49:04 am +02:00">Jul 10, 2019</time>_

---


the short code is: .replace(/<\/?[^>]+(>|$)  ;)